Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -757,4 +753,28 @@ public static String checkPrefixBeforeAppend(final String checkValue, final Stri
             return checkValue;
         }
     }
+
+    public static List<Object> checkPrefixInList(final List<Object> valueList, final String checkPrefix){
```

Comment:
These two methods can be combined.   No need to create a whole new method just to pass in a constant.